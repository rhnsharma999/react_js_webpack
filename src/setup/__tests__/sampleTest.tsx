import { shallow } from "enzyme"
import * as React from "react"

describe("this is a sample test", () => {
  const testComponent = (
    <div>
      <h1>Hello World</h1>
    </div>
  )
  const wrapper = shallow(testComponent)
  it("passes snapshot test", () => {
    expect(wrapper).toMatchSnapshot()
  })

  it("test find method", () => {
    expect(wrapper.find("h1")).toHaveLength(1)
  })

  it("test finding h1 tag", () => {
    expect(wrapper.find("h1").text()).toMatch("Hello World")
  })
})
