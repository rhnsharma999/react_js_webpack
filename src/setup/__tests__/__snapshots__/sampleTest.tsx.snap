// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`this is a sample test passes snapshot test 1`] = `
<div>
  <h1>
    Hello World
  </h1>
</div>
`;
